nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -130,7 +133,8 @@ def setup(self):
         os.chdir(self.args.workspace)
         fobs_initialize()
         AuthorizationService.initialize(EmptyAuthorizer())
-        AuditService.initialize(audit_file_name=WorkspaceConstants.AUDIT_LOG)
+        # AuditService.initialize(audit_file_name=WorkspaceConstants.AUDIT_LOG)
+        AuditService.the_auditor = SimulatorAuditor()
```

Comment:
Why do we need this `SimulatorAuditor` for?
Are we just mocking all its functionality?

Reply:
Simulator run server and clients in the same process. Each client / server share the same AuditService, which will write the audit log in a file. large number of concurrent clients will cause a file write error. Simulator does not need the audit log. SimulatorAuditor mock the function.